.eslintrc.js

```diff
@@ -2,6 +2,11 @@ module.exports = {
   extends: 'google',
   env: {
     node: true,
+    browser: true,
     es6: false
   },
+  rules: {
+    "no-var": "off",
+    "require-jsdoc": "off"
```

Comment:
These two rules were moved from .codeclimate.yml

Reply:
@eoinkelly This is such a sensible change. Will the README be updated to reflect the changes. Something like <scripts described> to run locally should be executed locally before pushing up a branch.

Reply:
I did add to the README about this https://github.com/ODNZSL/nzsl-online#running-the-tests - is that what you mean?

Reply:
Oh ok ! I'm a few steps behind.